### PR TITLE
combined similar steps as one in login feature

### PR DESCRIPTION
**Pull request by KarunAtreya**
In login feature, the step to logging with valid and invalid credentials are made to use same step.

### CODE REVIEW

**Review by PrajwolAmatya — APPROVED** [review]
lgtm

File: tests/e2e/features/login.feature [hunk 1]
```diff
@@ -10,7 +10,7 @@ Feature: login feature
 
 
     Scenario: user logs in with valid credentials
-        When the user logs in with valid username "user" and valid password "aaaa1234"
+        When the user logs in with username "user" and password "aaaa1234"
         Then the user should be navigated to the search page
         And the user should be displayed the message "Successfully signed in as user."
```
Comment (grgprarup) [inline tests/e2e/features/login.feature]:
```suggestion
        And the user should see the message "Successfully signed in as user."
```

Comment (grgprarup) [reply tests/e2e/features/login.feature]: Similar change for line 20, `Then the user should be displayed the error message "<message>"` to `Then the user should see the error message "<message>"`

File: tests/e2e/features/login.feature [hunk 2]
```diff
@@ -10,7 +10,7 @@ Feature: login feature
 
 
     Scenario: user logs in with valid credentials
-        When the user logs in with valid username "user" and valid password "aaaa1234"
+        When the user logs in with username "user" and password "aaaa1234"
         Then the user should be navigated to the search page
```
Comment (grgprarup) [inline tests/e2e/features/login.feature]:
```suggestion
        Then the user should be navigated to the search page
```
The step seems weird, after login user redirects to the search page.

Comment (KarunAtreya) [reply tests/e2e/features/login.feature]: you are right it should be homepage instead.

**Review by grgprarup — APPROVED** [review]
LGTM

**Pull request merged**